fixpatches/src/commonMain/kotlin/com/saveourtool/sarifutils/adapter/SarifFixAdapter.kt

```diff
@@ -1,13 +1,24 @@
+@file:Suppress(
+    "FILE_IS_TOO_LONG",
+    "FILE_UNORDERED_IMPORTS",  // false positive
```

Comment:
Do we have issue for this in diktat?

Reply:
Not a false positive, just misconfiguration.

Addressed with #47.